templates/tracker.mustache

```diff
@@ -148,18 +148,37 @@
                     tbody.children[i+1].appendChild(emptyNode);
                     break;
                 }
-                textnode = document.createTextNode(attempts[attempt_id]&&"Proctor link"||""); 
-
+                textnode = document.createTextNode(attempts[attempt_id]&&"Proctor link"||"");
                 node = document.createElement("a");    
                 node.setAttribute("title", "Proview Link");
                 node.setAttribute("href", attempts[attempt_id]&&attempts[attempt_id].proview_url||"");
                 node.setAttribute("target", "proview");
-                node.addEventListener("click",()=>{                                             {{! Event to display I frame containing Proview admin }}
-                    table.setAttribute("style","display: none");
-                    modal.setAttribute("style","display: block");
+                node.addEventListener("click",()=>{
+                const proctor_type = attempts[attempt_id].proctor_type;
+                     if(proctor_type=='ai_proctor'||proctor_type=='record_and_review'||proctor_type=='live_proctor'){
```

Comment:
@rohansharmasitoula you can instead do `['ai_proctor', 'record_and_review', 'live_proctor'].includes(proctor_type)`. It offers the same functionality and makes code more radable